### DISCUSSION TITLE
Help Docs

### DISCUSSION THREAD

**Discussion by nodlek-ctrl** [NaitLee/Cat-Printer#60] [Ideas]
So, I think this is a good use of this space.

I understand that adding documentation can be a cumbersome task, which is why I want to offer my thoughts and hear the opinions of other collaborators.
NaitLee, what did you have in mind for the docs? I understand that bloating the readme with documentation of all the nitty-gritty features is not ideal. So my question is; 
Do you prefer that we use the GitHub wiki, or just a .md in the project root with usage of the scripts, or something else entirely?
I have pretty well gathered a decent knowledge of the program (except ipp, I might need some help on that) so I would be happy to make a start either way.

**Comment by NaitLee** [comment]
I did a bit progress months ago and have put something [here](https://github.com/NaitLee/Cat-Printer/tree/helps-finally-coming/www/helps). More about it in #55.

I didn’t note GitHub Wiki at that time. I see it’s okay, and it may probably come with a way that also respects offline use (i.e. clone it as git repository, and integrate the docs to a release, view it in many ways.)

I’d investigate this thing later :) And I may make Dash XML <=> Markdown for this purpose.

That said, starting to write is encouraged — they will be distributed sooner.

**Reply by nodlek-ctrl** [reply]
Ok then. I read through your writeup on Dash, and it is surely creative. For now, I will gradually take notes and build up some documentation in md since I am somewhat familiar with it!
Thanks.

**Comment by NaitLee** [comment]
Public modification of this repo’s GitHub wiki is allowed.

There shouldn’t be much trouble as for now. Feel free to write something :)
